shared/chat/conversation/input/index.js.flow

```diff
@@ -28,4 +28,19 @@ export type Props = {
   typing: Array<Types.Username>,
 }
 
+export type MentionHocProps = {
```

Comment:
don't quite understand the names here, and also `MentionHocProps` in `mention-handlers.hoc.js`, which is a different type (!). Perhaps comments / different names would help?

Reply:
ah yeah, the intention is that `MentionHocProps` here are the props on the input that come from the mention hoc, whereas `MentionHocProps` in `mention-handlers.hoc.js` are the props the hoc takes from the rest of the container. I somehow missed the duped name heh, I'll clear it up